rqd/rqd/rqmachine.py

```diff
@@ -824,7 +824,7 @@ def reserveHT(self, frameCores):
         # Prefer to assign cores from the same physical cpu.
         # Spread different frames around on different physical cpus.
         avail_cores = {}
-        avail_cores_count = 0
+        avail_procs_count = 0
```

Comment:
Is it a good idea to rename from cores to procs? There's already the confusion with threads/cores...

Reply:
I agree the terminology on this method is confusion, but the this variable represents procs, and not cores in the context of the logic.

Reply:
> Is it a good idea to rename from cores to procs? There's already the confusion with threads/cores...

It seems both [cores](https://www.opencue.io/docs/concepts/glossary/#cores) and [procs](https://www.opencue.io/docs/concepts/glossary/#proc) have reserved terms in the glossary.

Reply:
This seems to raise some confusion, here's what I pieced together when working on this and reading about CPUs :

tldr: A "proc" is a logical core, a "core" is a physical core on the CPU.

In a mono/multi-threaded CPU, 1 core == 1 proc.
In a hyper-threaded CPU, 1 core == 2 procs.
Then comes hybrid CPUs with a mix of both, P-cores are hyper-threaded and E-cores are mono-threaded.
When listing the procs (logical cores), we first have the hyper-threaded ones, then the mono-threaded at the end.

A system can have multiple CPU, like with servers, each one with a physical id.

Using the CPU dump from [_cpuinfo_i9_12900_hybrid_ht_24-24-1-1](https://github.com/AcademySoftwareFoundation/OpenCue/pull/1296/commits/56ee4871af2165848274a4da165cf0572409666e)

| phys_id |  core_id 	| proc_id	|
|---	|---	|---	|
| 0   	|   0	|   0	|
| 0  	|   0	|   1	|
| 0	|   4	|   2	|
| 0	|   4	|   3	|
| 0	|   8	|   4	|
| 0	|   8	|   5	|
| ...	|   ...	|   ...	|

To illustrate CPUs, cores and procs:
![CPU](https://github.com/user-attachments/assets/39ac9753-d93f-45fc-b77a-78b9b4eabbd0)